starter-code/routes/celebrities.js

```diff
@@ -0,0 +1,79 @@
+const express = require("express");
+const router = express.Router();
+const Celebrity = require("../models/Celebrity.model");
+
+router.get("/", (req, res) => {
+  Celebrity.find().then((celebritiesFound) => {
+    console.log("Here are your celebrities: ", celebritiesFound);
+    res.render("celebrities/index", { celebrities: celebritiesFound });
+  });
+});
+
+router.get("/new", (req, res) => {
+  console.log("Visiting the page with the form to add a celeb");
+  res.render("celebrities/new");
+});
+
+router.post("/new-celebrity", (req, res) => {
```

Comment:
is good practice to have the same url on the get (that renders the form) and post (that gathers the data) request. So:
`router.post("/new",`